lib/auto_attach.py

```diff
@@ -31,6 +31,9 @@
 )
 from uaclient.files import state_files
 
+upro_logger = logging.getLogger("ubuntu_pro")
+LOG = logging.getLogger("ubuntu_pro.lib.auto_attach")
```

Comment:
Why do we need two distinct loggers here ?

Reply:
I am configuring the pro root logger via `upro_logger` since we will call other parts of the code which will be logging. The other logger is for logging in the file.